### PR TITLE
update main.go

### PR DESCRIPTION
**Pull request by angelafunk**
not using templates yet, so deleted template variable

### CODE REVIEW

File: appengine/gophers/gophers-2/main.go [hunk 1]
```diff
@@ -21,6 +16,4 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 		http.Redirect(w, r, "/", http.StatusFound)
 		return
 	}
-
-	indexTemplate.Execute(w, nil)
```
Comment (broady) [inline appengine/gophers/gophers-2/main.go]: This means the page will be blank. We still want to render the template, even though there aren't any parameters.

It's the same as writing out the index file.

Comment (broady) [reply appengine/gophers/gophers-2/main.go]: Or is this because you added a static handler for the index page?

Comment (angelafunk) [reply appengine/gophers/gophers-2/main.go]: static handler! WDYT?

Comment (broady) [reply appengine/gophers/gophers-2/main.go]: Sounds good.

This looks OK.

File: appengine/gophers/gophers-2/main.go [hunk 2]
```diff
@@ -6,11 +6,6 @@ import (
 
 	"google.golang.org/appengine"
 )
-
-var (
-	indexTemplate = template.Must(template.ParseFiles("index.html"))
-)
-
```
Comment (broady) [inline appengine/gophers/gophers-2/main.go]: Add back one of these newlines

**Pull request merged**